src/utils/__tests__/astFromValue.js

```diff
@@ -0,0 +1,265 @@
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import astFromValue from '../astFromValue';
+import {
+  GraphQLEnumType,
+  GraphQLInputObjectType,
+  GraphQLList
+} from '../../type/definition';
+import { GraphQLFloat } from '../../type/scalars';
+
+
+describe('astFromValue', () => {
+
+  it('converts boolean values to ASTs', () => {
+    expect(astFromValue(
```

Comment:
nit: for this whole file most of these could be just two lines. astFromValue(...) could be one line. and to.deep.equal(...) could be one line